static/js/planner.js

```diff
@@ -148,11 +148,11 @@ function changeViewDate(size, amount) {
                     view_end_date: end_date
                 },
                 success: function(data, textStatus, jqXHR) {
+                    $('#events-in-week-view').empty().append(data);
                     loadEventUpdateModal();
```

Comment:
Make sure to load all the modals after the append, so every html event element is affected.